nf_core/pipelines/lint/files_unchanged.py

```diff
@@ -173,6 +174,12 @@ def _tf(file_path: Union[str, Path]) -> Path:
                             shutil.copy(_tf(f), _pf(f))
                             passed.append(f"`{f}` matches the template")
                             fixed.append(f"`{f}` overwritten with template file")
+                        elif f.name in ["LICENSE", "LICENSE.md", "LICENCE", "LICENCE.md"]:
+                            # Report LICENSE as a warning since we are not using the manifest.author names
+                            # TODO: Lint the content of the LICENSE file except the line containing author names
```

Comment:
Is this TODO still valid?

Reply:
yes :) we would have to lint for the text of the licensev (except copyright line) and make this a fail if it doesn't match. But I'm leaving this for the next version.